content/03.methods.md

```diff
@@ -83,19 +83,19 @@ We used GATK Haplotype caller.
 #### SNV and INDEL calling
 
 We used four variant callers to call SNVs and INDELS from targeted DNA panel, WXS, and WGS data: Strelka2, Mutect2, Lancet, and VarDict.
-The input interval BED files for both panel and WXS data were padded by 100 bp on each side for all variant calling algorithm runs.
-The BED files for WGS were not padded for Mutect2 and Strelka2 runs, were padded by 300 bp on each side for Lancet, and by 100 bp on each side for VarDict.
+The input interval BED files for both panel and WXS data, derived from probes used to create the assays, were padded by 100 bp on each side for all variant calling algorithm runs.
+We utilized the non-padded BROAD Institute interval calling list `wgs_calling_regions.hg38.interval_list`, comprised of the full genome minus N bases, as the WGS input BED, unless otherwise noted below. 
```

Comment:
```suggestion
For WGS calling, we used the non-padded BROAD Institute interval calling list `wgs_calling_regions.hg38.interval_list`, comprised of the full genome minus N bases, as the WGS input BED, unless otherwise noted below. 
```

Is there a link to this file we can add?

Reply:
updated in upcoming commit